sdk/eventhub/azure-eventhub/tests/livetest/synctests/test_send.py

```diff
@@ -179,3 +179,53 @@ def test_send_with_create_event_batch_with_app_prop_sync(connstr_receivers):
             received.extend(r.receive_message_batch(timeout=5000))
         assert len(received) >= 1
         assert EventData._from_message(received[0]).properties[b"raw_prop"] == b"raw_value"
+
+
+@pytest.mark.liveTest
+def test_send_list(connstr_receivers):
+    connection_str, receivers = connstr_receivers
+    client = EventHubProducerClient.from_connection_string(connection_str)
+    payload = "A1"
+    with client:
+        client.send_batch([EventData(payload)])
+    received = []
+    for r in receivers:
+        received.extend([EventData._from_message(x) for x in r.receive_message_batch(timeout=10000)])
+
+    assert len(received) == 1
+    assert received[0].body_as_str() == payload
+
+
+@pytest.mark.liveTest
+def test_send_list_partition(connstr_receivers):
+    connection_str, receivers = connstr_receivers
+    client = EventHubProducerClient.from_connection_string(connection_str)
+    payload = "A1"
+    with client:
+        client.send_batch([EventData(payload)], partition_id="0")
+        message = receivers[0].receive_message_batch(timeout=10000)[0]
+        received = EventData._from_message(message)
+    assert received.body_as_str() == payload
```

Comment:
may be worth validating the failure mode of the incorrectly aligned partition/session scenario as part of one of these tests?

Reply:
We clarified offline that EH doesn't have session